Tasks/AzureResourceGroupDeploymentV2/task.json

```diff
@@ -14,8 +14,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 217,
-        "Patch": 1
+        "Minor": 219,
```

Comment:
First things first, let's bump it to 220 according to [versioning guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md); since we passed cut-off date.

Reply:
Ok, I just didn't know which week it was.